### PR TITLE
Adding Java coming soon as a supported driver

### PR DESCRIPTION
**Pull request by dalanmiller**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ Or, get started right away with our ten-minute guide in these languages:
 * [**JavaScript**](http://rethinkdb.com/docs/guide/javascript/)
 * [**Python**](http://rethinkdb.com/docs/guide/python/)
 * [**Ruby**](http://rethinkdb.com/docs/guide/ruby/)
+* **Java** [(Coming Soon!)](https://github.com/rethinkdb/rethinkdb/issues/3930)
```
Comment (danielmewes) [inline README.md]: I think this might look better in lower case: "(coming soon)"

**Comment by danielmewes** [comment]
Added one comment. Good idea!

**Comment by dalanmiller** [comment]
Changed to lower case, feel free to merge if you're cool with it @danielmewes

**Comment by danielmewes** [comment]
Looks good. Merged.

**Pull request merged**